docs/advanced_usage/hooks.md

```diff
@@ -41,12 +41,12 @@ Checking that the hook is now in use is as simple as attempting to pass a [Solut
 
 ```python
 dodgy_batch = problem.generate_batch(10)  # Generate a SolutionBatch of size 10
-dodgy_batch.access_values()[1, 1] = float('inf')  # Inject a rogue inf value
+dodgy_batch.access_values()[1, 1] = float("inf")  # Inject a rogue inf value
 problem.evaluate(dodgy_batch)  # Pass the SolutionBatch to the problem to evaluate
 ```
 
 ???+ abstract "Output"
-    ```python
+    ```bash
```

Comment:
For output boxes like these, perhaps we could go with raw text regions, instead of bash regions?
```suggestion
    ```
```

Reply:
Sure!